### ISSUE TITLE
Change Password

### ISSUE THREAD

**Issue by Idontker** [amosproj/amos2022ss04-digital-identity#130]
## User story

1. As a HR employee
2. I want / need to change my password
3. So that I can define a new password to login securly

## Acceptance criteria

- page which contains a formular to change the password: 
  - input field to enter the email  
  - input field to enter your current password 
  - input field to enter the new password
  - input field to confirm the new password
  - submit button
- passwords should be disguised unless a toggle button is clicked
- submit should check whether new password and the confirm password are matching
- submit should check whether the new password is sufficently secure.

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
